src/Presentation/GraphQL/Base/Mutation.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Types;
+
+namespace Presentation.GraphQL.Base
+{
+    public class Mutation : ObjectGraphType
+    {
+        public Mutation()
+        {
+            Name = "Mutation";
+            Field(typeof(StringGraphType), "BasicMutation",null, null, x => "Mutation");
```

Comment:
Should we add a description instead of the first null field ?

Reply:
Done